core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -147,7 +152,14 @@ public InetSocketAddress getRpcAddress() {
   public void start() throws Exception {
     LOG.info("Starting...");
     mJournalSystem.start();
-    mJournalSystem.gainPrimacy();
+    try {
+      mJournalSystem.gainPrimacy();
+    } catch (Throwable t) {
```

Comment:
Could we update the docs on this? Does it only backup when shutting down Alluxio? what about error in journal replay? or does it assume that will trigger a shutdown?

Reply:
I've added a note in the docs.